source/common/ssl/context_impl.cc

```diff
@@ -391,8 +391,10 @@ ServerContextImpl::ServerContextImpl(ContextManagerImpl& parent, Stats::Scope& s
            int encrypt) -> int {
           ContextImpl* context_impl = static_cast<ContextImpl*>(
               SSL_CTX_get_ex_data(SSL_get_SSL_CTX(ssl), sslContextIndex()));
-          return dynamic_cast<ServerContextImpl*>(context_impl)
-              ->sessionTicketProcess(ssl, key_name, iv, ctx, hmac_ctx, encrypt);
+          ServerContextImpl* server_context_impl = dynamic_cast<ServerContextImpl*>(context_impl);
+          RELEASE_ASSERT(server_context_impl != nullptr);
```

Comment:
this will crash on the following line. why?

Reply:
This one is specifically to fix Coverity's requirement that the return value of `dynamic_cast` be checked for null-ness.